posts/2023-07-24-leadership-council-update.md

```diff
@@ -0,0 +1,62 @@
+
+We discussed and approved [RFC 3355](https://rust-lang.github.io/rfcs/3355-rust-spec.html), which establishes that we would like to have a Rust specification and that we would like the Foundation to hire an editor for the specification (note that this RFC does not actually approve any aspects of that specification yet, only starting the work to create the specification).
+
+And we've been discussing all of this work in the [public council stream on Zulip](https://rust-lang.zulipchat.com/#narrow/stream/392734-council) if you'd like to follow along or contribute to the discussion.
+
+## Looking forward with your feedback
+
+We have a lot of ideas on problems of the Rust Project we _could_ work on, but we know we, as a council, can only realistically make progress on a few topics at the same time.
+
+We've come up with a proposed list of our first three priorities, and we'd like the project's feedback on whether this matches your expectations. This is a starting place for iterative improvement over time; we're planning on re-evaluating whether these priorties are still the right ones about every two months or so.
+
+The type of feedback we're most interested in is hearing about topics or issues you think are more important or urgent than one or more of the following three priorities, and why we should tackle that topic instead of what we have listed here. If this list sounds good to you, we'd like to hear that as well!
+
+Each team representative on the council will be reaching out to their team specifically for feedback on this list, but we also welcome feedback from the Rust community in this Zulip topic (TODO link) or via email to <council@rust-lang.org>.
+
+The priorities we're proposing we concentrate on first are:
+
+1. Documenting and improving processes for interaction with the Rust Foundation including formalising the role of project directors in this process (the trademark policy process may be part of this).
+2. Establishing plans for how to strengthen the moderation team (i.e., 2 is not a sustainable number and we currently have no contingent moderators) and strengthen moderation policies.
+3. Figuring out the "shape" of the Rust Porject. The Leadership Council needs to reflect the will of the Rust Project so it is important to know who that includes.
+
+If you're interested in the items under consideration that we think aren't as high priority as these, you can read the full, unsorted list [in this document](https://hackmd.io/@rust-leadership-council/Skdo7VKK3).
+
+We are also interested in delegating other work that the council won't have bandwidth to take on; if you have proposals for something you'd like to work on which will eventually need the council's support/approval, please reach out!
```

Comment:
@ehuss I know we're not ready to announce any process for surfacing items to the Council's backlog, but I wonder if we'd like to mention that we're working towards doing that?

Reply:
If you want to mention something, that's fine with me. I'm not sure there is much to share at this point, though, other than we are thinking about it. If we make progress before the next update, then we can share something more substantive then.

Reply:
I decided not to address this since we're not ready yet.